internal/cluster/validator.go

```diff
@@ -224,6 +224,40 @@ func (v *clusterValidator) isApiVipValid(c *clusterPreprocessContext) Validation
 	return boolValue(err == nil)
 }
 
+func (v *clusterValidator) isNetworkTypeValid(c *clusterPreprocessContext) ValidationStatus {
+	validNetworkTypes := []string{models.ClusterNetworkTypeOVNKubernetes, models.ClusterNetworkTypeOpenShiftSDN, models.ClusterCreateParamsNetworkTypeAutoAssign}
+	if !funk.ContainsString(validNetworkTypes, swag.StringValue(c.cluster.NetworkType)) && c.cluster.NetworkType != nil {
+		return ValidationFailure
```

Comment:
Since we have a valid default pending is not suitable here.  Pending should be used when we are waiting for information such as inventory, or hosts were not discovered yet.